buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -425,7 +425,12 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
                 echo "The '${name}' test suite will be excluded"
                 continue
             }
-            def TEST_FLAG = (name.contains('+')) ? name.substring(name.indexOf('+')+1).toUpperCase() : ''
+            // Add a TEST_FLAG to a specific target if one was passed with the test target. Eg. sanity.functional+aot
+            if (name.contains('+')) {
+                def temp_test_flag = name.substring(name.indexOf('+')+1).toUpperCase()
+                TEST_FLAG = (TEST_FLAG) ? TEST_FLAG + ',' + temp_test_flag : temp_test_flag
```

Comment:
This seems to imply that the test pipelines will be the same ones as the non-jit spec.  If thats the case it will make triaging failures more difficult for the pipelines that are used by multiple specs.

Reply:
See https://github.com/eclipse/openj9/issues/7782#issuecomment-557689426

Reply:
Also, explain why you believe it will be more difficult

Reply:
I believe Joe is referring to the fact that Jenkins build history will be interleaved with regular, cmake, and jitserver specs. Which makes it more difficult for finding trends. I'm not sure if/how TRSS handles this. IMO it's up to the triagers (Pete) to decide if this is a problem.

Reply:
Are the parent jobs different?  If so (and as mentioned in the comment link shared above), both Jenkins and TRSS can give a triager the view of their own child jobs separate from the other test jobs.

Reply:
The parent jobs/runs are different, and come from different builds/job-names. 
I.e 
[Pipeline_Build_Test_JDK8_x86-64_linux](https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK8_x86-64_linux/) 
and
[Pipeline_Build_Test_JDK8_x86-64_linux_cm](https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK8_x86-64_linux_cm/)
will both launch
[Test_openjdk8_j9_sanity.functional_x86-64_linux_Nightly](https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.functional_x86-64_linux_Nightly/).
Agree from the parent you can see downstream test status. The issue is there is no Jenkins view to see test suite trend for a specific spec once we intermingle more than 1 spec per build. Whether that is actually an issue is up to the triagers. Again, I don't know if/how TRSS handles this.
Ie. From this [view](https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.functional_x86-64_linux_Nightly/), we would have runs of default xlinux, some as jitserver and some as cmake.